rules/hardcoded_credentials.go

```diff
@@ -79,12 +81,24 @@ func (r *credentials) matchAssign(assign *ast.AssignStmt, ctx *gosec.Context) (*
 					}
 				}
 			}
+
+			// Match the value assigned
+			for _, e := range assign.Rhs {
+				if val, err := gosec.GetString(e); err == nil {
```

Comment:
These deep nestings can be quite hard to follow. Can this be flattened? Maybe just `continue` on `err != nil`, then put the rest of the logic under? Currently, this `err` seems to be unhandled. If we truly don't care about it, you can `_` it, though you'll likely want to log it or return it